plone/app/discussion/browser/moderation.pt

```diff
@@ -88,7 +88,8 @@
                     <tbody>
                         <tal:block repeat="item batch">
                             <tr class="commentrow"
-                                tal:define="even repeat/item/even"
+                                tal:define="even repeat/item/even;
+                                   email python:getattr(item.getObject(), 'author_email')"
```

Comment:
As the view is paginated I guess this is a better compromise rather than adding metadata to the catalog. Thanks for coming up with this solution!

One minor nitpick: would you mind aligning ``email`` with ``even``, that's the convention being used *everywhere* in templates when you have multiple statements on a ``tal:define``. This way it is easy to see what are the variables being defined.